

o fix documentation
o use $(src) in Makefile (fixes cross-compilation)

Both spottet by Geert Uytterhoeven

Signed-off-by: Jörn Engel <joern@wohnheim.fh-wedel.de>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/Makefile              |    2 +-
 25-akpm/scripts/checkstack.pl |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -puN Makefile~small-fixes-to-checkstack Makefile
--- 25/Makefile~small-fixes-to-checkstack	2004-06-28 17:40:52.384493576 -0700
+++ 25-akpm/Makefile	2004-06-28 17:40:52.392492360 -0700
@@ -1103,7 +1103,7 @@ endif #ifeq ($(mixed-targets),1)
 .PHONY: checkstack
 checkstack:
 	$(OBJDUMP) -d vmlinux $$(find . -name '*.ko') | \
-	$(PERL) scripts/checkstack.pl $(ARCH)
+	$(PERL) $(src)/scripts/checkstack.pl $(ARCH)
 
 # FIXME Should go into a make.lib or something 
 # ===========================================================================
diff -puN scripts/checkstack.pl~small-fixes-to-checkstack scripts/checkstack.pl
--- 25/scripts/checkstack.pl~small-fixes-to-checkstack	2004-06-28 17:40:52.386493272 -0700
+++ 25-akpm/scripts/checkstack.pl	2004-06-28 17:40:52.390492664 -0700
@@ -13,7 +13,7 @@
 #	M68k port by Geert Uytterhoeven and Andreas Schwab
 #
 #	Usage:
-#	objdump -d vmlinux | stackcheck_ppc.pl [arch]
+#	objdump -d vmlinux | stackcheck.pl [arch]
 #
 #	TODO :	Port to all architectures (one regex per arch)
 
_
